src/styles/blocks/calendar.scss

```
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-days-gap;
+
+  max-width: $calendar-max-width;
+  padding: $calendar-pading-y $calendar-pading-x;
+
+  &__day {
+    height: $calendar-day-size;
+    width: $calendar-day-size;
+    background-color: #eee;
+    border: 1px solid black;
+
+    font-size: 30px;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    transition: 0.5s;
```

you should specify the properties that will change
```suggestion
    transition-duration: background-color 0.5s, transform 0.5s;
```

---

agree, you should specify the properties that will change, do it as Yegor suggested